internal/archive/archive.go

```diff
@@ -34,11 +34,16 @@ func readAsTar(src, basePath string, uid, gid int, mode int64, writeFn func(tw *
 	go func() {
 		var err error
 		defer func() {
-			w.CloseWithError(err)
+			_ = w.CloseWithError(err)
```

Comment:
apologies for the nitpick, but if we aren't using any of the return values I think we should omit the assignment to `_`